BUILD.md

```diff
@@ -341,56 +333,78 @@ index 7dfa97c..6d99e71 100644
 
  set_property(TARGET LLVMSupport PROPERTY LLVM_SYSTEM_LIBS "${system_libs}")
 ```
+   * Linux
+   Download llvm source code 6.0.1 and unzip to /llvm/source/path, then install to /llvm/install/path
+   ```
+   cd /llvm/source/path
+   mkdir build
+   cd build
+   cmake .. -DLLVM_TARGETS_TO_BUILD=X86 -DCMAKE_BUILD_TYPE=Release
+   cmake --build.
```

Comment:
>cmake --build. [](start = 3, length = 14)

please change to "make -j$(nproc)". Otherwise it builds with single thread and could be very slow.